app/controllers/profile_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ProfileController < ApplicationController
+  before_filter :authenticate_user!, only: [ :create, :update, :destroy ]
+  respond_to :json
+  def index
+    #return recipes's current_user.
```

Comment:
before you return recipe for the current user you must test if user signed in : 
 if user_signed_in?
  #return user recipes
, and use named scope